_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+					<date-start>Jan 9</date-start>
+				</text>
+				<text color='black'>
+					<unclear>19--</unclear>
+				</text>
+				<text color='light-blue'>
+					<title>Little Women - Little Women Married</title>
+				</text>
+				<text color='blue'>
+					<date-end>Feb 4</date-end>
+				</text>
+				<text color='light-blue'>
+					<title>Mr Norris Changes Trains</title>
+				</text>
+				<text color='blue'>
+					<date-end>Feb 4</date-end>
+				</text>
+				<text color='black'>
+					<date-start>Nov 3 1946</date-start>
+					<title>A Serial Universe</title>
+					<note>returned</note>
+					<note>(Composition
+						<member-name>Gertrude Stein</member-name>
+						. gave copy to
+						<member-name>R W</member-name>)
+					</note>
+					<date-start>June 20 1946</date-start>
+					<title>The Negro in our History</title>
+					<title>Proletarian Literature</title>
+				</text>
+			</borrowing-record>
+		</side>
+		<side type="reverse"></side>
+	</div>
+	<div type="lending-library-card">
+		<side type="front">
+			<heading>
+				<text color="purple">
+					<member-name>Ellen Wright</member-name>
+				</text>
+				<text color="black">
+					<address>14 rue M. le Prince</address>
+				</text>
+			</heading>
+			<borrowing-record>
+				<text color="purple">
+					<date-start>Sept 19 1949</date-start>
+					<title>The Pasquier Chronicles</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26</date-end>
+				</text>
+				<text color="purple">
+					<strikethrough>
+						<title>The Close</title>
+					</strikethrough>
+					<title>Christine</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26 1949</date-end>
+				</text>
+				<text color="purple">
+					<title>Lafcadio's Adventures</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26 1949</date-end>
+					<date-start>Oct 26 1949</date-start>
+					<title>The Gambler</title>
+					<date-start>Oct 26</date-start>
+					<title>Walter Rathenau his life &amp; work by
+						
+						<author>Count Harry Kessler</author>
+						<publication-info>
+							<publisher>Gerald Howe Ltd</publisher>
+							<address>23 Soho Square London</address>
+							<year>1929</year>
+						</publication-info>
+					</title>
+				</text>
+				<text color="faded-blue">
+					<note>
+						<unclear>Also wants	
+							<author>Christina</author>
+						</unclear>
+						&amp;	
+						<author>Morand's</author>
+						Books
+					</note>
+				</text>
+			</borrowing-record>
+		</side>
+		<side type="reverse"/>
+	</div>
+</div>
+```
+
+### Organizing Information: Categories and Hierarchy
+In order to categorize the information, I created a root tag that nested the separate cards. Within a div element for each card, I separated the encoding further with a ```<heading>``` element and a ```<borrowing-record>``` element. This first element contains Wright's name/address, while the second contains the remaining information on the card, including specific borrowed books and notes. 
```

Comment:
Please edit the ``` (triple tick marks) here to ` (single tick marks) so that the code snippets here show up inline as opposed to in a block. (The triple tick marks are good for blocks of code, the single tick marks are good for inline code.)